pandas/core/indexes/datetimes.py

```diff
@@ -175,6 +175,8 @@ def _new_DatetimeIndex(cls, d):
 class DatetimeIndex(DatelikeOps, TimelikeOps, DatetimeIndexOpsMixin,
                     Int64Index):
     """
+    Store timestamps using NumPy’s datetime64 dtype.
+
     Immutable ndarray of datetime64 data, represented internally as int64, and
     which can be boxed to Timestamp objects that are subclasses of datetime and
     carry metadata such as frequency information.
```

Comment:
For the message in your validation error, I'd say the parameters are documented in the wrong order. It could be a problem in the script, but can you take a look and double check please?

Reply:
sure, I was to query as I looked over and found they were all in

Reply:
how about `infer_dst` that is .. deprecated:: 0.15.0. ?

Reply:
If it's not in the signature (I don't think it is), you can get rid of it.